crates/abi/abi/HEVM.sol

```diff
@@ -5,6 +5,8 @@ struct DirEntry { string errorMessage; string path; uint64 depth; bool isDir; bo
 struct FsMetadata { bool isDir; bool isSymlink; uint256 length; bool readOnly; uint256 modified; uint256 accessed; uint256 created; }
 struct Wallet { address addr; uint256 publicKeyX; uint256 publicKeyY; uint256 privateKey; }
 struct FfiResult { int32 exitCode; bytes stdout; bytes stderr; }
+struct AccountAccess { address accessor; address account; uint256 kind; bool initialized; uint256 oldBalance; uint256 newBalance; bytes deployedCode; uint256 value; bytes data; bool reverted; StorageAccess[] storageAccesses; }
```

Comment:
Can we include the enum definition in this file and that instead of `uint256 kind`?

Reply:
Ah interesting, should be ok if we can't use the enum here since we can still use in `Vm.sol` in forge-std, so will defer to you + @mattsse / @Evalir here about the abigen issue

Reply:
don't worry about it, crates/abi is deprecated and will be removed soon. It's currently only used for trace/log decoding